tests/ts_cluster_capabilities_SUITE.erl

```diff
@@ -182,10 +185,15 @@ sql_select_downgrade_a_node_test(Config) ->
     rt:wait_until_capability(Node_A, ?SQL_SELECT_CAP, v2),
     rt:wait_until_capability(Node_B, ?SQL_SELECT_CAP, v2),
     rt:wait_until_capability(Node_C, ?SQL_SELECT_CAP, v2),
+
+    %% pending changes to how capabilities are communicated around the ring, this
+    %% section will expect v2. Once capabilities are changed in a future version of riak
+    %% (or if testing 1.4.0 as current)
```

Comment:
You could add a link to the issue tracking the upcoming change: https://bashoeng.atlassian.net/browse/RTS-1415